src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -533,6 +533,7 @@ private static void BuildRegionSize(PatchRegion region)
             {
                 region.Width += offset;
                 region.Height += offset;
+                region.Columns = region.Rows = 1;
```

Comment:
Does the columns and rows actually need to be stored like this? I get the impression that before the region patch positions was implicitly known based on the size, couldn't the same thing be done still (i.e. don't add the rows and columns variables but calculate them from the width and height of the region)?